src/clr-angular/popover/signpost/providers/signpost-focus-manager.service.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class SignpostFocusManager {
+  private _triggerEl: HTMLElement;
+
+  set triggerEl(value: HTMLElement) {
+    this._triggerEl = value;
```

Comment:
nit: I don't see value in this setter, why not just have a public property?

Reply:
It's not passing el from one class to another so it doesn't need to be public. This `_triggerEl` is only for internal use of the service.

Reply:
The setter is public (and when it compiles down its public), and even if you aren't ever getting it from elsewhere its overhead to privatize it IMO. Its a 'nit'.

Reply:
No that's why it's private and has setter because it makes it obvious that the element shouldn't be read from outside.